src/components/ui/tenantList.tsx

```diff
@@ -0,0 +1,56 @@
+import { Avatar } from '@/components/ui/avatar';
+import { Ellipsis, Trash2, SquarePen, User } from 'lucide-react';
+import {
+  DropdownMenu,
+  DropdownMenuTrigger,
+  DropdownMenuContent,
+  DropdownMenuGroup,
+  DropdownMenuItemWithIcon,
+} from '@/components/ui/dropdown-menu';
+
+interface TenantListProps {
+  name: string;
+}
+
+/**
+ * @example
+ * ```tsx
+ * return (
+ * <TenantList name='Momo'/>
+ * )
+ */
+const TenantList = ({ name }: TenantListProps) => {
+  return (
+    <div className="flex items-center justify-between w-full h-full mx-5 my-3">
+      <div>
+        <Avatar>
+          <User />
+        </Avatar>
+      </div>
+      <div className="ml-4">{name}</div>
+      <div className="ml-auto">
+        <DropdownMenu>
+          <DropdownMenuTrigger asChild>
+            <button className="p-4">
+              <Ellipsis />
+            </button>
+          </DropdownMenuTrigger>
```

Comment:
Is it possible to remove the purpose border that appears when I click the ellipsis button? I believe it's not necessary for this component.
<img width="94" alt="image" src="https://github.com/Tascurator/tascurator-frontend/assets/124953279/4ae33222-e050-4060-ab59-1c8bdfd1bb22">

https://github.com/Tascurator/tascurator-frontend/assets/124953279/232f8910-87ff-491c-b380-852ccdca4ccc

Reply:
I'm checking this issue right now and just a quick question, is your PC's accent colour purple?

Reply:
solved:)